pkg/kv/kvserver/loqrecovery/storage.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package loqrecovery
+
+import (
+	"io"
+	"io/fs"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/loqrecovery/loqrecoverypb"
+	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
+	"github.com/cockroachdb/errors"
+	"github.com/cockroachdb/pebble/vfs"
+)
+
+const RECOVERY_PLAN_DIR = "recovery-plans"
```

Comment:
nit: constants should be camelcase. We should also prefix this with `loq-`, to differentiate it from the recovery Pebble does on startup.